### DISCUSSION TITLE
General Enquires on WD

### DISCUSSION THREAD

**Discussion by sivaponting** [ExpediaGroup/waggle-dance#292] [General]

**Comment by sivaponting** [comment]
Hi @patduin ,

Like hive metastore, does the WD also stores info about mappings in separate metastore?

**Comment by patduin** [comment]
No all info is part of the configuration, no state is stored. There is one exception where the configuration can be written back to disk but that's not an extremely useful feature and I would consider that deprecated. For all practical purposes WD doesn't hold state.

**Comment by sivaponting** [comment]
Hi @patduin,
Jus want to understand the no. of connections to metastore. 
In my use case I have 1 primary metastore & 1 remote metastore.
Whenever I do the sql query. The total current connections to all metastores is getting increased.
Ex.,
If I do "show databases", total current connections to all metastores: 2
For the "describe table", total current connections to all metastores: 4

Is this the expected behavior in WD?
